### ISSUE TITLE
Ctrl-C needs to be used twice to interrupt

### ISSUE THREAD

**Issue by blueyed** [boxed/mutmut#38]
```
2. Checking mutants
⠇ 7/282  🎉 3  ⏰ 0  🤔 0  🙁 4^C
Aborted!
^CException ignored in: <module 'threading' from '/usr/lib/python3.7/threading.py'>
Traceback (most recent call last):
  File "/usr/lib/python3.7/threading.py", line 1273, in _shutdown
    t.join()
  File "/usr/lib/python3.7/threading.py", line 1032, in join
    self._wait_for_tstate_lock()
  File "/usr/lib/python3.7/threading.py", line 1048, in _wait_for_tstate_lock
    elif lock.acquire(block, timeout):
KeyboardInterrupt
```

mutmut 1.0.0.

**Comment by blueyed** [comment]
I've waited several seconds before the 2nd one.

**Comment by boxed** [comment]
I've seen some surprisingly long wait times before it actually quits. Haven't had time to look at it but I'm guessing it's waiting for the sub process to complete?

**Comment by boxed** [comment]
I think this is fixed with 1.0.1 which should be on pypi in a few seconds. I just had to set the daemon flag on a thread. If I'm wrong please reopen.
